data/monster/lua/test.lua

```diff
@@ -0,0 +1,50 @@
+-- a testing layout
+local test = NewMonsterType('Test')
```

Comment:
@Crypton33 are right, it should create a new MonsterType if "Test" doesn't exist